server/routes/creatingLicences/handlers/additionalLicenceConditions.test.ts

```diff
@@ -8,14 +8,12 @@ import ConditionService from '../../../services/conditionService'
 const conditionService = new ConditionService(null) as jest.Mocked<ConditionService>
 const licenceService = new LicenceService(null, null, null, conditionService) as jest.Mocked<LicenceService>
 
-jest.spyOn(conditionService, 'getGroupedAdditionalConditions').mockReturnValue(
-  Promise.resolve([
-    {
-      category: 'group1',
-      conditions: [{ text: 'Condition 1', code: 'condition1', category: 'group1', requiresInput: false }],
-    },
-  ])
-)
+jest.spyOn(conditionService, 'getGroupedAdditionalConditions').mockResolvedValue([
```

Comment:
you usually don't need to manually create spies, you can add:  

`jest.mock('../../../services/conditionService')` 

after the import and it means if your instantiated instances will be mocked and you can just call:

```
conditionService.getGroupedAdditionalConditions.mockResolvedValue(...)
```